src/integrations/automation/types.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * An interface for an automation service (Retool, Zapier, etc)
+ */
+export interface IAutomationService<Args, Ret> {
+  run(args: Args): Promise<Ret>;
+}
+
+export type Automation = 'roaster';
```

Comment:
This should be union type in the future? eg. when we add another one it will be `'roaster' | 'fylla'`?

Reply:
Exactly

Reply:
We usually use enums more so might be better to just use enum, or is there some other reason?

Reply:
Just copied my feed version implementation. Can do enum if you prefer, I don't mind tbh

Reply:
I would prefer enum yes, since you can also use it as value for different checks vs union is just type.

Reply:
Done!